data/profiles.json

```diff
@@ -142,5 +142,30 @@
       "https://x.com/umbristol",
       "https://umbracocommunity.social/@umbristol"
     ]
+  },
+   {

```

Comment:
Entries need to be sorted alphabetically by alias. Can you please move these profiles into the appropriate position alphabetically please.